ava.config.js

```diff
@@ -15,4 +14,4 @@ export default {
 	nonSemVerExperiments: {
 		"configurableModuleFormat": true
 	  },
-}
\ No newline at end of file
+}
```

Comment:
You (accidentally) committed a docs/.DS_Store file.

Can you add it to .gitignore and remove the file?

Reply:
done